src/modules/map/components/CompareSlider.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div class="compare-slider">
+        <font-awesome-icon
+            class="compare-slider-caret-left"
+            :icon="['fas', 'caret-up']"
+            :rotation="270"
+        />
+        <div class="compare-slider-line"></div>
+
+        <font-awesome-icon
+            class="compare-slider-caret-right"
+            :icon="['fas', 'caret-up']"
+            :rotation="90"
+        />
+    </div>
+</template>
+
+<script>
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+
+export default {
+    components: {
+        FontAwesomeIcon,
+    },
+}
+</script>
+
+<style lang="scss" scoped>
+@import 'src/scss/webmapviewer-bootstrap-theme';
```

Comment:
I might be good trying to "factorize" your CSS a bit.
Meaning trying to find common rules to what is below, and apply them together.
Something like

```
.compare-slider,
.compare-slider-caret-left,
.compare-slider-caret-right {
    position: absolute
    z-index: $zindex-compare-slider;
}
```

Aslo, little hint in SCSS, you can declare child directly in the parent if their name follow a cascading convention (exactly what you did here)
```
.compare-slider {
    // ...

    &-caret-left {
        // this will be applied to compare-slider-caret-left
    }
}